src/js/components/uploadDetachedFiles/UploadDetachedFileValidation.jsx

```diff
@@ -341,6 +343,10 @@ class UploadDetachedFileValidation extends React.Component {
 				validationButton = <button className='pull-right col-xs-3 us-da-disabled-button' disabled>You do not have permissions to publish</button>;
 			}
 		}
+
+		if (this.state.error !== 0) {
```

Comment:
Why was this moved down here instead of being left in the else if block?

Reply:
the logic above it is for if it is not publishing. It would change states before so this was the simplest solution I saw